src/System.Windows.Forms/src/System/Windows/Forms/ContextMenuStrip.cs

```diff
@@ -119,5 +119,35 @@ protected override void SetVisibleCore(bool visible)
 
             base.SetVisibleCore(visible);
         }
+
+        protected override void OnOpened(EventArgs e)
+        {
+            if (IsHandleCreated && !IsInDesignMode)
+            {
+                AccessibilityNotifyClients(AccessibleEvents.SystemMenuPopupStart, -1);
+
+                if (IsAccessibilityObjectCreated)
+                {
+                    AccessibilityObject.RaiseAutomationEvent(Interop.UiaCore.UIA.MenuOpenedEventId);
```

Comment:
Consider adding `using static Interop;` so that it would be more consistent:
```suggestion
                    AccessibilityObject.RaiseAutomationEvent(UiaCore.UIA.MenuOpenedEventId);
```
Same in `ToolStripDropDownItem`